### PR TITLE
Creates create-organization-profile page

### PR DESCRIPTION
**Pull request by alexjmcm**
### PR Checklist:

Hey there! It's great to have you here. Here's a quick checklist to help you make a shiny PR:

* [ ] Have you checked out the guidelines in our [Contributing](https://github.com/FightPandemics/FightPandemics/blob/master/CONTRIBUTING.md) document?
* [ ] Have you looked if there might be other open [Pull Requests](https://github.com/FightPandemics/FightPandemics/pulls?q=is%3Apr+is%3Aopen+sort%3Aupdated-desc) for the same update/change?

### Quick Description of the PR

What does it aim to achieve, and why?

### An Overview of the Changes

* A few notes on your commits.
* To help understand them.

### CODE REVIEW

File: client/src/components/Button/SubmitButton.js [hunk 1]
```diff
@@ -1,54 +1,26 @@
...
-import BaseButton from "./BaseButton";
-import { theme } from "constants/theme";
-const { button } = theme;
-
-// props: inline, primary, primarylight, secondary, tertiary
-
-const SubmitButton = styled(BaseButton).attrs(({ size, inline }) => {
-  return { size, inline };
-})`
-  ${button.regular}
```
Comment (snowiesuet) [inline client/src/components/Button/SubmitButton.js:11]: Why are these removed?

File: client/src/pages/CreateOrganizationProfile.js [hunk 2]
```diff
@@ -0,0 +1,340 @@
...
+  const renderError = (field) => {
+    if (errors.includes(field) && (!formData[field] || !formData[field].length))
+      return errorMsg[field];
+  };
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    populateErrors();
+    if (!errors.length) {
+      console.log(name);
```
Comment (snowiesuet) [inline client/src/pages/CreateOrganizationProfile.js:160]: Please remove console.log's here

File: backend/.env.example_docker [hunk 3]
```diff
@@ -1,10 +1,9 @@
 PORT=8000
-MONGO_URI=mongo/fightpandemics
```
Comment (snowiesuet) [inline backend/.env.example_docker:2]: Not sure if this file is meant to be edited? @robinv85

**Comment by snowiesuet** [comment]
Hey @alexjmcm thank you for the pull request but for future reference:

Please read [this](https://github.com/FightPandemics/FightPandemics/blob/master/README.md#important-notes-and-considerations) before making a pull request in the future. There are too much changes in one single commit. The branch should be in `/feature`, more [here](https://www.notion.so/fightpandemics/How-to-create-a-ticket-and-pull-request-on-GitHub-3ee2abb2d7f24b90ac35553dd0a117c4#4af978185b8d4320b45e2f546785c42d)

File: backend/.env.example_docker (hunk 3 above)
Comment (anniegiang) [reply backend/.env.example_docker:2]: @snowiesuet Good catch! 

@alexjmcm This file should NOT be edited. The purpose of this file was to set an outline on how to create a `backend/.env` file. 

I also noticed that you deleted `backend/.env.example_local` and `client/.env.example`. There was no need to delete them. Please properly follow the setup in the README on how to add environment variables.

File: client/src/components/Button/SubmitButton.js (hunk 1 above)
Comment (anniegiang) [reply client/src/components/Button/SubmitButton.js:11]: @snowiesuet @alexjmcm There have been changes to this file prior to this PR. Please always pull and rebase with master and work with the latest changes.

File: client/src/components/Typography/Title/CustomH1.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
+import styled from "styled-components";
```
Comment (anniegiang) [inline client/src/components/Typography/Title/CustomH1.js:1]: This file was removed.

File: client/src/pages/AirTableCOVID.js [hunk 5]
```diff
@@ -0,0 +1,79 @@
+import React, { Component } from "react";
```
Comment (anniegiang) [inline client/src/pages/AirTableCOVID.js:1]: This file will be removed.

**Comment by anniegiang** [comment]
duplicate of #399
